From 5b80ca39b1f01177e98c78bbc622dfda6f7a7e71 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Mon, 29 Sep 2014 08:27:15 +0200
Subject: [PATCH 2/4] pcap-config: mitigate multilib conflict

libdir path is different on 64bit and 32bit arches. Hence when installing both
multilib versions on the system yum complains about conflicting pcap-config
file.

Hence remove libdir references from pcap-config, libdir is in dynamic linker
path anyway.
---
 pcap-config.in | 27 ++++++++-------------------
 1 file changed, 8 insertions(+), 19 deletions(-)

diff --git a/pcap-config.in b/pcap-config.in
index 206be3b..75f2c9f 100644
--- a/pcap-config.in
+++ b/pcap-config.in
@@ -7,7 +7,6 @@
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 includedir="@includedir@"
-libdir="@libdir@"
 V_RPATH_OPT="@V_RPATH_OPT@"
 LIBS="@LIBS@"
 
@@ -36,16 +35,6 @@ do
 	esac
 	shift
 done
-if [ "$V_RPATH_OPT" != "" ]
-then
-	#
-	# If libdir isn't /usr/lib, add it to the run-time linker path.
-	#
-	if [ "$libdir" != "/usr/lib" ]
-	then
-		RPATH=$V_RPATH_OPT$libdir
-	fi
-fi
 if [ "$static" = 1 ]
 then
 	#
@@ -54,16 +43,16 @@ then
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir -lpcap $LIBS"
+		echo "-lpcap @LIBS@"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir $LIBS"
+		echo "@LIBS@"
 	elif [ "$show_cflags" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "-L$libdir -lpcap $LIBS"
+		echo "-lpcap @LIBS@"
 	elif [ "$show_additional_libs" = 1 ]
 	then
 		echo "$LIBS"
@@ -75,15 +64,15 @@ else
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir $RPATH -lpcap"
+		echo "-lpcap"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_cflags" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "-L$libdir $RPATH -lpcap"
+		echo "-lpcap"
 	fi
 fi
-- 
1.8.3.1

